src/renderer/components/SearchInput/styles.module.css

```diff
@@ -0,0 +1,54 @@
+:root {
```

Comment:
The style guide I've tried to follow here looks like that:

### General

- Prefer `padding-top`, `padding-bottom`, `padding-left etc. over `padding` to avoid bugs through order-critical dependencies between properties, except when you only need `padding` or `border` etc.
- Try to avoid `!important` as much as possible (goal is to get rid of them altogether at one point)

### Naming

- Class names and variable names are always camelCase
- Sub-classes can have short, local names (like `.active` or `.warning`)

### Ordering

- Properties are sorted alphabetically
- Order of groups: variables (`--borderRadius`), properties (`height: 10px;`), pseudo elements (`::before`), sub-classes (`.active`), always separated with a newline
- Classes (`.searchInput` etc.) are also always separated with a newline

### Variables

- Variables which are used across multiple classes in the same file are introduced at the beginning of the document, inside a `:root` group. Every variable name is prefixed with `--local-` (for example `--local-borderRadius: 10px;`)
- Global variables (used across files) are as-is (later we can think about prefixing them with `--theme-` but who knows ..)
- Variables only used inside of a single class can be named however, they are always defined at the beginning of the class and separated with a newline from the following properties

Reply:
I don't know yet how theming comes into all of this, still learning about the theming system